go.sum

```diff
@@ -184,12 +184,14 @@ github.com/envoyproxy/go-control-plane v0.10.1/go.mod h1:AY7fTTXNdv/aJ2O5jwpxAPO
 github.com/envoyproxy/protoc-gen-validate v0.1.0/go.mod h1:iSmxcyjqTsJpI2R4NaDN7+kN2VEUnK/pcBlmesArF7c=
 github.com/envoyproxy/protoc-gen-validate v0.6.2/go.mod h1:2t7qjJNvHPx8IjnBOzl9E9/baC+qXE/TeeyBRzgJDws=
 github.com/evanphx/json-patch v4.5.0+incompatible/go.mod h1:50XU6AFN0ol/bzJsmQLiYLvXMP4fmwYFNcr97nuDLSk=
-github.com/evanphx/json-patch v4.9.0+incompatible h1:kLcOMZeuLAJvL2BPWLMIj5oaZQobrkAqrL+WFZwQses=
 github.com/evanphx/json-patch v4.9.0+incompatible/go.mod h1:50XU6AFN0ol/bzJsmQLiYLvXMP4fmwYFNcr97nuDLSk=
+github.com/evanphx/json-patch v4.11.0+incompatible h1:glyUF9yIYtMHzn8xaKw5rMhdWcwsYV8dZHIq5567/xs=
```

Comment:
> __[Secret Scan]__ It may be dangerous to commit the AWS secret access key. ([view](https://sider.review/gh/repos/237181535/pulls/212#issue-6525398))
>
> <details>
>
> AWS secret access key is an important credential that allows you to use AWS programmatically.
> Once this leaks, an attacker can access and operate your AWS resources.
>
> See also:
> * https://docs.aws.amazon.com/general/latest/gr/aws-access-keys-best-practices.html
>
> | Rule | Severity |
> | ---- | -------- |
> | `sider.secrets.aws.secret_access_key` | `warning` |
>
> You can [close this issue](https://sider.review/gh/repos/237181535/pulls/212#issue-6525398) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

I closed this issue. Reason:
- When this is not an AWS secret access key.